run_service.sh

```diff
@@ -1067,6 +1096,7 @@ export STOP_TRADING_IF_STAKING_KPI_MET=true
 export RESET_PAUSE_DURATION=45
 export MECH_WRAPPED_NATIVE_TOKEN_ADDRESS=$WXDAI_ADDRESS
 export MECH_CHAIN_ID=ethereum
+export OMEN_SUBGRAPH_URL="https://gateway.thegraph.com/api/$SUBGRAPH_API_KEY/subgraphs/id/9V1aHHwkK4uPWgBH6ZLzwqFEkoHTHPS7XHKyjZWe8tEf"
```

Comment:
You will need to provide this for all the subgraphs:

- NETWORK_SUBGRAPH_URL (gnosis blocks)
- OMEN_SUBGRAPH_URL
- TRADES_SUBGRAPH_URL (omen)
- REALITIO_SUBGRAPH_URL

Reply:
@Adamantios missing from your list the subgraph for conditonal tokens IMO

Reply:
Correct! This is the `CONDITIONAL_TOKENS_SUBGRAPH_URL`.